app/services/rfd.remote.server.ts

```diff
@@ -104,9 +105,27 @@ export async function getRemoteRfds(rfdClient: Api): Promise<RfdWithoutContent[]
   return handleApiResponse(result)
 }
 
-export async function searchRfds(user: User | null, q: string): Promise<SearchResults> {
+export async function searchRfds(
+  user: User | null,
+  params: IterableIterator<[string, string]>,
+): Promise<SearchResults> {
   const rfdClient = client(user?.token || undefined)
-  const result = await rfdClient.methods.searchRfds({ query: { q } })
+  const query: SearchRfdsQueryParams = { q: '' }
+
+  for (let [k, v] of params) {
+    switch (k) {
+      case 'q':
+        query.q = v
+      case 'attributes_to_crop':
+        query.attributesToCrop = v
+      case 'highlight_pre_tag':
+        query.highlightPreTag = v
+      case 'highlight_post_tag':
+        query.highlightPostTag = v
+    }
+  }
```

Comment:
where are the snake case ones coming from? can they just be camel all the time from a client POV?

Reply:
Good point. They are coming from the `Search.tsx` component. At this point it is only due to history. I'll update them.